cli/remotebazel/remotebazel.go

```diff
@@ -385,9 +385,9 @@ func getBaseBranchAndCommit(remoteData string) (branch string, commit string, er
 		}
 		branch = defaultBranch
 
-		defaultBranchCommitHash, err := getHeadCommitForRemoteBranch(remoteData, defaultBranch)
+		defaultBranchCommitHash, err := getHeadCommitForLocalBranch(defaultBranch)
```

Comment:
nit: above this a few lines we are running "git rev-parse HEAD" which could be changed to `getHeadCommitForLocalBranch("HEAD")`

Reply:
Done